pkg/cache/v3/linear.go

```diff
@@ -378,6 +419,29 @@ func (cache *LinearCache) computeResponse(watch watch, replyEvenIfEmpty bool) (W
 	return watch.buildResponse(resources, removedResources, returnedVersions, responseVersion), nil
 }
 
+func (cache *LinearCache) notifyAllForNode(nodeID string) error {
```

Comment:
It should not be called `notifyAll` as only notifying some specific watches based on the nodes. Maybe `notifyNode`?